File: 02_drawGraph/app/src/main/java/com/yanheng/drawgraph/MainActivity.java
package com.yanheng.drawgraph;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.yanheng.drawgraph.graph.DrawViewDemo;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        initView();
    }

    private void initView() {
        Button button = findViewById(R.id.button);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                setGraph();
            }
        });
    }

    private void setGraph() {
        DrawViewDemo drawViewDemo = (DrawViewDemo)findViewById(R.id.draw_view);
//        drawViewDemo.
    }
}

File: 06_KotlinMyApplication/app/src/main/java/com/yanheng/myapplication/MainActivity.kt
package com.yanheng.myapplication

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        L.d()
    }

    override fun onStart() {
        super.onStart()

    }
}
